docs/src/examples/components/Dialog/Content/DialogExampleFooter.shorthand.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, Dialog, Box } from '@stardust-ui/react'
+import * as React from 'react'
+
```

Comment:
Can we also add screener tests for this? Just place `DialogExampleFooter.shorthand.steps.ts` in the same directory:

```ts
import getScreenerSteps from '../commonScreenerSteps'

const config: ScreenerTestsConfig = {
  steps: getScreenerSteps(),
}

export default config
```